tests_metricflow/snapshots/test_name_edge_cases.py/str/DuckDB/test_filter_by_metric_name_with_2_measures_from_same_source_node__query_output.txt

```diff
@@ -0,0 +1,127 @@
+test_name: test_filter_by_metric_name_with_2_measures_from_same_source_node
```

Comment:
This snapshot is new in this commit because the test would error before the prior commit.